### PR TITLE
CIV-12408 DRH SDO

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-12408

### Change description ###
Web capture_13-2-2024_9460_xui-civil-ccd-pr-4005 preview platform hmcts net

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToComplexTypes/CreateSDO-SDOR2-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/CaseEventToComplexTypes/CreateSDO-SDOR2-nonprod.json:1]: Should the name be: CreateSDO-SDO-R2-nonprod.json ?

File: ccd-definition/FixedLists/SdoR2/SmallClaimsMethod-SDO-R2-nonprod.json
Comment (ShwetaTandel-hmcts) [inline ccd-definition/FixedLists/SdoR2/SmallClaimsMethod-SDO-R2-nonprod.json:1]: In Person -- P is not capital in UI/Figma

File: ccd-definition/FixedLists/SdoR2/PhysicalTrialBundleOptions-SDO-R2-nonprod.json
Comment (ShwetaTandel-hmcts) [inline ccd-definition/FixedLists/SdoR2/PhysicalTrialBundleOptions-SDO-R2-nonprod.json:1]: Physical hearing bundle in No and Party on figma

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SDO-R2-nonprod.json
Comment (ShwetaTandel-hmcts) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SDO-R2-nonprod.json:1]: sdoR2SmallClaimsUploadDocToggle is missing legal advisor permiission

**Review by ShwetaTandel-hmcts — COMMENTED** [review]
Added a few comments

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

File: ccd-definition/CaseEventToComplexTypes/CreateSDO-SDOR2-nonprod.json
Comment (bhagyashreesharma90) [reply ccd-definition/CaseEventToComplexTypes/CreateSDO-SDOR2-nonprod.json:1]: Done

File: ccd-definition/CaseEventToComplexTypes/LengthListOtherSdoR2SmallClaims-SdoR2-nonprod.json
[the remark this answers is not in the document]
Comment (bhagyashreesharma90) [reply ccd-definition/CaseEventToComplexTypes/LengthListOtherSdoR2SmallClaims-SdoR2-nonprod.json:1]: Done

File: ccd-definition/FixedLists/SdoR2/SmallClaimsMethod-SDO-R2-nonprod.json
Comment (bhagyashreesharma90) [reply ccd-definition/FixedLists/SdoR2/SmallClaimsMethod-SDO-R2-nonprod.json:1]: Done

File: ccd-definition/FixedLists/SdoR2/PhysicalTrialBundleOptions-SDO-R2-nonprod.json
Comment (bhagyashreesharma90) [reply ccd-definition/FixedLists/SdoR2/PhysicalTrialBundleOptions-SDO-R2-nonprod.json:1]: Done

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SDO-R2-nonprod.json
Comment (bhagyashreesharma90) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-SDO-R2-nonprod.json:1]: Done

**Pull request merged**